playbooks/ansible-deployment-example.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: get awx-operator git repo
+  ansible.builtin.git:
+    repo: "https://github.com/ansible/awx-operator.git"
+    dest: "~/awx-operator"
+    version: "{{ awx_operator_version }}"
+    force: true
+
+- name: deploy kustomization file
+  copy:
+    content: |
+      ---
+      apiVersion: kustomize.config.k8s.io/v1beta1
+      kind: Kustomization
+      resources:
+        # Find the latest tag here: https://github.com/ansible/awx-operator/releases
+        - github.com/ansible/awx-operator/config/default?ref={{ awx_operator_version }}
+      
+      # Set the image tags to match the git version from above
+      images:
+        - name: quay.io/ansible/awx-operator
+          newTag: {{ awx_operator_version }}
+      
+      # Specify a custom namespace in which to install AWX
+      namespace: {{ awx_namespace }}
+    dest: "~/awx-operator/kustomization.yaml"
+    mode: 0640
+
+- name: Deploy controller to the K8s cluster specified in ~/.kube/config.
+  shell: kustomize build . | kubectl apply -f -
+  environment:
+    PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/snap/bin"
+  args:
+    chdir: "~/awx-operator"
+  changed_when: true
```

Comment:
You could use [Kustomize lookup plugin](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/kustomize_lookup.html) here

Reply:
Thanks, implemented. Did not know that one.